draft-ietf-webpush-protocol.xml

```diff
@@ -453,9 +453,15 @@ Location: https://push.example.net/d/qDIYHNcfAIPP_5ITvURr-d6BGtYnTRnk
 ]]></artwork>
       </figure>
       <t>
-        A push service MAY generate a 413 (Payload Too Large) status code in
-        response to requests that include an entity body that is too large.
-        Push services MUST NOT generate a 413 status code in responses to an
+        A push service MAY return a 429 (Too Many Requests) status code <xref target="RFC6585"/>
+        when an application server has exceeded its rate limit for push message
+        delivery. The push service SHOULD also include a Retry-After header <xref target="RFC7231"/>
+        to indicate how long the application server must wait before it makes another request. 

```

Comment:
"to indicate how long the application *is requested to* wait before making another request"  Using lowercase "must" just confuses some people.

Then there is the question of scope: does this apply to the endpoint, or to the entire server as a whole.  RFC 6585 doesn't answer that question, and I don't think that we should either, but we might suggest something.  Maybe just add that the scope of the 429 status code is determined by the push service and may apply globally, on an application server basis, or for a specific URL.